formcentral/test.yaml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+  build_base_image:
+    image: ${APP_NAME}_base
+    build:
+      context: .
+      dockerfile: Dockerfile.base
+    command: sh -c 'exit 0'
+  main:
+    depends_on:
+      - cockroachdb
+    image: ${APP_NAME}_dev
+    build:
+      args:
+        - APP_NAME=${APP_NAME}
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      CHATBASE_DATABASE: chatroach
+      CHATBASE_HOST: cockroachdb
+      CHATBASE_PORT: 26257
+      CHATBASE_USER: root
+    command: |
+      bash -c '
+        tools/wait-for-it.sh cockroachdb:26257 -t 60
+        cat /migrations/* > /tmp/merged-latest.sql
```

Comment:
I forgot where we ended in our discussion, but I seem to remember we got rid of this, no? Removed the logic of migrating the database from the app itself and put it in the database? Or as a separate container that "inits" the database?